pipeline-modules/app-service/azure/java-unit-test.yaml

```diff
@@ -0,0 +1,24 @@
+provisioner: azure
+name: java-unit-test
+version: 1
+revision: 1
+displayName: Run JAVA UnitTest with Maven
+description: run JAVA unit test with Maven
+target: ""
+keywords:
+  - java
+author: CloudCover
+meta: {}
+inputs:
+  properties:
+    working_dir:
+      title: Working directory
+      description: The current working directory to execute command
+      type: string
+      default: repo
+template: |
+  steps:
+  - script: mvn test
```

Comment:
Could add a template variable named test-args? and append it in front of `mvn test` command? 
like:
mvn test {% for arg in test_args %} {{ arg }} {% endfor %}
this way we can accommodate all below cases
mvn test
# Run a single test class.
$ mvn -Dtest=TestApp1 test
# Run multiple test classes.
$ mvn -Dtest=TestApp1,TestApp2 test

# Run a single test method from a test class.
$ mvn -Dtest=TestApp1#methodname test

# Run all test methods that match pattern 'testHello*' from a test class.
$ mvn -Dtest=TestApp1#testHello* test

# Run all test methods match pattern 'testHello*' and 'testMagic*' from a test class.
$ mvn -Dtest=TestApp1#testHello*+testMagic* test

Reply:
@pranaySinghDev  this is done.  Please review again.